mu4/scenes/notation/view/notationpaintview.cpp

```diff
@@ -279,6 +276,15 @@ void NotationPaintView::mouseMoveEvent(QMouseEvent* ev)
     m_inputController->mouseMoveEvent(ev);
 }
 
+void NotationPaintView::mouseDoubleClickEvent(QMouseEvent* event)
+{
+    forceActiveFocus();
```

Comment:
Why not setFocus(true) as in mousePressEvent?

Reply:
It won't work, only "forceActiveFocuse" will grab the real focus